utilities/is-05-control/is05Control.py

```diff
@@ -113,21 +113,41 @@ def configure_receiver(url, sender_id, sdp_data):
 
     send_request(url, body)
 
+def get_request(url):
```

Comment:
The function naming seems confusing.

I think `send_request` is "\<verb\>\_\<noun\>"  
but `get_request` is "\<adjective\>\_\<noun\>"

I think spelled out, they could be more clearly `send_patch_request` and `send_get_request`?